base/src/main/java/com/smartdevicelink/transport/TransportConstants.java

```diff
@@ -90,6 +90,8 @@
     public static final String BIND_REQUEST_TYPE_STATUS = "BIND_REQUEST_TYPE_STATUS";
     public static final String BIND_REQUEST_TYPE_USB_PROVIDER = "BIND_REQUEST_TYPE_USB_PROVIDER";
 
+    public static final String EXTRA_CONNECTED_VEHICLE_INFO = "extra_connected_vehicle_info";
```

Comment:
The constant define in proposal is as below. Please change it.
`public static final String VEHICLE_INFO = "vehicle_info";`